packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -68,6 +69,7 @@ export interface BaseBoxProps
   outline?: OutlineProperty<string>;
   float?: FloatProperty;
   willChange?: WillChangeProperty;
+  textDecoration?: TypographyProps['textDecoration'];
```

Comment:
This is a bigger change than it looks, because we're now saying we can style text decoration on box when we previously thought no text styles should be applied on box. 🤔